src/components/accession2/edit/CalculatorModal.tsx

```diff
@@ -150,6 +150,7 @@ export default function CalculatorModal(props: CalculatorModalProps): JSX.Elemen
               onChange={(value) => onChange('subsetCount', value)}
               type='text'
               value={record.subsetCount}
+              disabledCharacters={[',', '.']}
```

Comment:
do we need '-' here and the other place?

Reply:
Yes, we probably don't want negatives quantities haha